packages/fxa-auth-server/package.json

```diff
@@ -95,18 +98,28 @@
     "poolee": "^1.0.1",
     "punycode.js": "2.1.0",
     "qrcode": "^1.4.4",
+    "react": "^16.13.1",
```

Comment:
I think we should consider not using React here. As far as I can tell it's only being used in fxa-auth-server to facilitate mounting the Storybook stories (a component that uses a hook to load the contents). I think we could accomplish something similar with a plain JS fetch call and insert the returned HTML into the plain HTML story ([@storybook/html](https://www.npmjs.com/package/@storybook/html)). This would allow us to remove/no longer add `react`, `react-dom`, `@storybook/react`, `@types/react`, and `@types/react-dom` as dependencies. @LZoog thoughts?

Reply:
Yeah I think it would be safe to move all the new react deps to devDependencies. I'm still a little unsure about it since it's a **lot** of additional dependencies to look after (keep updated) for a single Storybook component, where React is not being used for anything else in fxa-auth-server.

Reply:
I think we can at least uninstall `@storybook/addon-links` and `@storybook/addon-actions`, I don't think we'll need either of those for this.

I definitely get your point though. @Yogita98 what would be the level of effort to see what this looks like with `@storybook/html`, would it take more than half a day? I don't think you should spend more than that here since this is working, but I'm curious what it would look like in comparison to the modified `.stories` file which [shows two different states for `cadReminder`](https://github.com/Yogita98/fxa/compare/state-previewing-solution...LZoog:state-previewing-solution#diff-721989f388164241c41b2b3cc4cd438e1aa0c53f2fd2866bbbd655ad9285739cR35-R45). 🤔

Reply:
@Yogita98 I got a version working on my end using only @storybook/html - Gist is here: https://gist.github.com/jodyheavener/46dec26d85f1f567742f17d9730c12b0 (note that I renamed `RenderedEmail.tsx` to `storybook-email.ts`)

Reply:
_Sweet_, after seeing that, @Yogita98 I think we should move forward with the non-React version to eliminate the extra dependencies. You can take what Jody did there and combine it with my directory structure updates and showing two stories for `cadReminder` ([handy link](https://github.com/Yogita98/fxa/compare/state-previewing-solution...LZoog:state-previewing-solution)).